src/lib/features/feature-toggle/feature-toggle-strategies-store.ts

```diff
@@ -514,6 +514,126 @@ class FeatureStrategiesStore implements IFeatureStrategiesStore {
         };
     }
 
+    // WIP copy of getFeatureOverview to get the search PoC working
+    async searchFeatures({
+        projectId,
+        userId,
+        queryString,
+    }: { projectId: string; userId?: number; queryString: string }): Promise<
+        IFeatureOverview[]
+    > {
+        let query = this.db('features');
+        if (projectId) {
+            query = query.where({ project: projectId });
+        }
+
+        if (queryString?.trim()) {
+            // todo: we can run cheaper query when no colon is detected
+            const tagQuery = this.db
+                .from('feature_tag')
+                .select('feature_name')
+                .whereRaw("(?? || ':' || ??) LIKE ?", [
+                    'tag_type',
+                    'tag_value',
+                    `%${queryString}%`,
+                ]);
+
+            query = query
```

Comment:
this is the most important part: search by string anywhere in feature name or anywhere in feature tags